src/lib/core/components/visualizations/implementations/LineplotVisualization.tsx

```diff
@@ -321,8 +349,14 @@ function LineplotViz(props: VisualizationProps) {
           ? {}
           : {
               numeratorValues: undefined,
+              // change denominatorValues following filteredProportionArray
               denominatorValues:
-                yAxisVar != null ? yAxisVar.vocabulary : undefined,
+                yAxisVar != null
+                  ? filteredProportionArray != null &&
```

Comment:
I think `filteredProportionArray` needs to be a dependency of the useCallback

Reply:
In my opinion, this is a sort of nice to have, not must have, because Ann's backend work would make it happen without throwing error at least.